tests/test_atom_client.py

```diff
@@ -0,0 +1,335 @@
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_listing_collections.yaml'))
+def test_listing_collections():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections()
+    assert len(collections) == 2
+    assert collections[0]['title'] == 'Top Fonds'
+    assert collections[0]['type'] == 'resource'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_collection_list.yaml'))
+def test_collection_list():
+    client = AtomClient(**AUTH)
+    collection_ids = client.collection_list('test-fonds')
+    assert len(collection_ids) == 2
+    assert collection_ids[0] == 'test-subfonds'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_note.yaml'))
+def test_rendering_record_containing_a_note():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections()
+    print str(collections)
+    assert len(collections) == 2
+    assert collections[0]['notes'][0]['content'] == 'Note content'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_find_collections_search.yaml'))
+def test_find_collections_search():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections(search_pattern='Test fonds')
+    print str(collections)
+    assert len(collections) == 1
+    assert collections[0]['title'] == 'Test fonds'
+    assert collections[0]['type'] == 'resource'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_find_collections_search_no_results.yaml'))
+def test_find_collections_search_no_results():
+    client = AtomClient(**AUTH)
+    no_results = client.find_collections(search_pattern='Nonexistent')
+    assert len(no_results) == 0
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_listing_collections_search_spaces.yaml'))
+def test_listing_collections_search_spaces():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections(identifier="2015044 Aa Ac")
+    assert len(collections) == 1
+    assert collections[0]['title'] == 'Resource with spaces in the identifier'
+    assert collections[0]['levelOfDescription'] == 'Fonds'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_listing_collections_sort.yaml'))
+def test_listing_collections_sort():
+    client = AtomClient(**AUTH)
+    asc = client.find_collections(sort_by='asc')
+    assert len(asc) == 2
+    assert asc[0]['title'] == 'Test fonds'
+    assert asc[0]['type'] == 'resource'
+
+    desc = client.find_collections(sort_by='desc')
+    assert len(desc) == 2
+    assert desc[0]['title'] == 'Top Fonds'
+    assert desc[0]['type'] == 'resource'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_find_component_parent_with_top_level_parent.yaml'))
+def test_find_component_parent_with_top_level_parent():
+    client = AtomClient(**AUTH)
+    type, id = client.find_parent_id_for_component('test-subfonds')
+
+    assert type == AtomClient.RESOURCE
+    assert id == 'test-fonds'
```

Comment:
`type_` and `id_` work, but so would more-descriptive `resource_type` and `resource_id`.  Shadowing builtins is problematic since you may try to access that builtin later and not realize you're getting the variable instead.  This is usually less of a problem in tests, but a good habit to be in.